metron-deployment/roles/metron_kafka_topics/defaults/main.yml

```diff
@@ -21,6 +21,7 @@ topics_to_create:
   - { topic: "bro",         num_partitions: 1, replication_factor: 1, retention_gb: 10 }
   - { topic: "yaf",         num_partitions: 1, replication_factor: 1, retention_gb: 10 }
   - { topic: "snort",       num_partitions: 1, replication_factor: 1, retention_gb: 10 }
+  - { topic: "asa", num_partitions: 1, replication_factor: 1, retention_gb: 10 }
```

Comment:
should we create the kafka topic if we aren't starting the sensor as part of the default set of sensors?  Shouldn't we handle this like squid, where we have the user create the topic if they set up the sensor?